### ISSUE TITLE
Execution Error - Unraid 7.0.0beta3

### ISSUE THREAD

**Issue by unabletoconnect2429** [SpaceinvaderOne/RetroNASinabox#12]
Hello,

I'm getting an error starting the VM under unraid 7beta3. Tried removing the vm config and letting it regenerate, same error as below. Any ideas?

Execution error

internal error: Child process (/usr/libexec/virtiofsd --print-capabilities) unexpected exit status 2: error: invalid value '' for '--fd ': cannot parse integer from empty string For more information, try '--help'.

**Comment by stry8993** [comment]
Not sure if yours is saying wrong file type, fstype btrfs, but I'm using btrfs on my cache drive and zfs on my main array, and both are unsupported. btrfs was supported on beta 2... so I don't know whats up.

**Comment by unabletoconnect2429** [comment]
Yes, Mine is doing the same error in the logs of the docker container.

**Comment by stry8993** [comment]
Well, I'm not glad its happening, but I am glad we've narrowed it down. I
know SpaceinvaderOne is a busy guy, but hopefully he gets around to this. I
do wonder though, if we might have to wait until v7 is out of beta, since
things keep changing.

**Comment by ProbablyCrazier** [comment]
If you edit the VM template and change the vdisk location to auto, and the vdisk bus to sata it resolves the issue and it boots properly.

**Comment by unabletoconnect2429** [comment]
I finally had a bit of time to look into this a bit more. Fix was easy.

Edit the config of the created VM

* Change machine to the latest version of Q35
* Change the disk image to SATA
* Edit the XML and ensure the disk type is qcow2 (mine was raw)

**Comment by ianwesty** [comment]
> I finally had a bit of time to look into this a bit more. Fix was easy.
> 
> Edit the config of the created VM
> 
> * Change machine to the latest version of Q35
> * Change the disk image to SATA
> * Edit the XML and ensure the disk type is qcow2 (mine was raw)

Thanks, this worked for me using Version 7 stable.
